test/orchard/java/resource_test.clj

```diff
@@ -8,6 +8,8 @@
 
 (deftest project-resources-test
   (testing "get the correct resources for the orchard project"
-    (let [resources (resource/project-resources)]
+    (let [resources (->> (resource/project-resources)
```

Comment:
What was wrong with this test before? There were more resources returned or something?

Reply:
>  There were more resources returned or something?

Yes especially in local envs were there can be more resources for arbitrary reasons.

Reply:
Got it.